rero_ils/modules/patrons/views.py

```diff
@@ -102,17 +104,34 @@ def profile(viewcode):
     patron = Patron.get_patron_by_user(current_user)
     if patron is None:
         raise NotFound()
+    if request.method == 'POST':
+        loan = Loan.get_record_by_pid(request.values.get('loan_pid'))
+        item = Item.get_record_by_pid(loan.get('item_pid'))
+        data = {
+            'item_pid': item.pid, 'pid': request.values.get('loan_pid'),
```

Comment:
can you put one key per line? It will be more readable.

Reply:
done